Products/ZenEvents/TrapFilter.py

```diff
@@ -232,8 +233,8 @@ def _parseV1FilterDefinition(self, lineNumber, action, remainingTokens):
                 filterDef.specificTrap = remainingTokens[1]
                 if filterDef.specificTrap != "*" and not filterDef.specificTrap.isdigit():
                     return "Specific trap '%s' invalid; must be non-negative integer" % filterDef.specificTrap
-            else:
-                return "Missing specific trap number or '*'"
+            elif len(remainingTokens) == 1:
+                oid = oidOrGenericTrap
```

Comment:
Why assign `oid` to `oidOrGenericTrap` again?  That happened on line 226 and hasn't had its value changed.

Additionally, removing the `return` statement says that having `len(remaingTokens) != 2` is OK now.  Is that correct?

Reply:
Idea behind removing return statement to allow using v1 with specific oid only, so yeah, it should be ok. Original checks are still in place, so in this case with 1 remaining token it would be specific or global oid